<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hedm_calibration_results_dialog</class>
 <widget class="QDialog" name="hedm_calibration_results_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1281</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HEDM Calibration Results</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="6">
    <widget class="QComboBox" name="grain_id">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QCheckBox" name="show_all_grains">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Show all grains</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="detector">
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="grain_id_label">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Grain ID:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="detector_label">
     <property name="text">
      <string>Detector:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <spacer name="horizontal_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="9">
    <layout class="QVBoxLayout" name="canvas_layout"/>
   </item>
   <item row="1" column="8">
    <widget class="QCheckBox" name="show_legend">
     <property name="text">
      <string>Show legend</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="9">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>detector</tabstop>
  <tabstop>show_all_grains</tabstop>
  <tabstop>grain_id</tabstop>
  <tabstop>show_legend</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>hedm_calibration_results_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>874</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>hedm_calibration_results_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>636</x>
     <y>874</y>
    </hint>
    <hint type="destinationlabel">
     <x>640</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
